src/models/reposyncstate.ts

```diff
@@ -4,6 +4,44 @@ import { merge } from "lodash";
 import { sequelize } from "models/sequelize";
 import { Config } from "interfaces/common";
 
+export interface RepoSyncState {
```

Comment:
I need this type in 2 of the other PRs I need to merge in. Figured it made more sense to add it in the base PR to keep the subsequent PRs a little on the lighter side (and saving y'all reviewing something twice).

Reply:
Why would we use Pick? Shouldn't we just change interface -> type and rename to RepoSyncStateType?

Reply:
Actually, maybe we leave it as an interface and rename it something like `RepoSyncStateProperties` and then update the class to implement this interface e.g. `export class RepoSyncState extends Model implements RepoSyncStateProperties` as it currently isn't typed.